mapping/sustainability.yaml

```
@@ -1336,17 +1327,17 @@
     mapping: |-
         Project-level:
 
-        1. Add a `Metric` object to the `.forecasts` array.
+        1. Add a [`Metric`](../../reference/schema.md#metric) object to the [`.forecasts`](project-schema.json,,forecasts) array.
```

```suggestion
        1. Add a [`Metric`](../../reference/schema.md#metric) object to the [`forecasts`](project-schema.json,,forecasts) array.
```

plus "Project-level" should be "Project level" above

---

Fixed in 73bc7abfdb6a4bf562298f1f80ac2d6cfd7656ec